HAL/pico/include/config_defaults.hpp

```diff
@@ -129,7 +142,7 @@ const Config default_config = {
         CommunicationBackendConfig {
             .backend_id = COMMS_BACKEND_CONFIGURATOR,
             .activation_binding_count = 1,
-            .activation_binding = { BTN_RT2 },
+            .activation_binding = { BTN_MB1 },
```

Comment:
This will be conflicting with the bootsel binding at the moment so can you revert the change to this line for now and it can be changed in a future PR